src/components/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import DayPicker, { DayModifiers, DayPickerProps } from 'react-day-picker';
+import 'react-day-picker/lib/style.css';
+import DatePickerWrapper from './DatePickerWrapper';
+
+type Props = {
+  onChange: (day: Date | undefined) => void;
+  initialDay?: Date;
+} & DayPickerProps;
+
+export const DatePicker: React.FC<Props> = ({ onChange, initialDay, ...props }: Props) => {
+  const [selectedDay, setSelectedDay] = React.useState(initialDay);
+
+  const handleDaySelect = (day: any, { selected }: DayModifiers) => {
+    setSelectedDay(selected ? undefined : day);
+  };
+
+  const handleChange = React.useCallback(
+    (date: Date | undefined) => {
+      onChange(date);
+    },
+    [onChange],
+  );
+
+  React.useEffect(() => {
+    handleChange(selectedDay);
+  }, [selectedDay]);
```

Comment:
missing dependency 'handleChange' is this intentional? If it is, disable the linter warning, if its not add the dependency